src/example.js

```diff
@@ -29,8 +29,11 @@ return {
     },
     '&:hover': {
       border: [
-        [1, 'solid', 'blue'],
-        [1, 'solid', 'green']
+        [1, 'solid', 'blue']
+      ],
+      boxShadow: [
+        [0, 0, 0, '10px', 'blue'],
```

Comment:
@kof I'ts just another "forgotten thing" between jss-default-unit and jss-expand. If we simply add:
`'box-shadow': 'px',`
on line https://github.com/cssinjs/jss-default-unit/blob/master/src/defaultUnits.js#L34 it will fix this issue.